Fix - Update SDK docs (#1047)
iotaledger/iota-wiki: next/docs/iota-sdk/docs/contribute.md

  - GitHub
---

# Contribute to the IOTA SDK

> Thank you for considering contributing to the IOTA SDK. We offer various ways for you to get involved and make a
> difference.

Reviewer: ```suggestion
If you're thinking about contributing to the IOTA SDK project, please consider the following ways to get involved.
```

## Join the IOTA Libraries Initiative

The [IOTA Libraries Initiative](https://github.com/iota-community/X-Team_IOTA_Libraries) is a collaborative effort aimed
at enhancing the developer experience. You can contribute in the following areas:

- Quality assurance and code review
- Documentation improvements

> - Creating code samples

Reviewer: Wrong voice.

```suggestion
- Code sample creation
```

To get started, join the #experience channel on [Discord](https://discord.iota.org).

## Contribute to the Project's GitHub Repository

> All the project's code is open source and hosted on [GitHub](https://github.com/iotaledger/iota-sdk). You can contribute
> in the following ways:

Reviewer: ```suggestion
All of the project's code is open source and hosted on [GitHub](https://github.com/iotaledger/iota-sdk). You can contribute
in the following ways:
```
